flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/HiveParserDMLHelper.java

```diff
@@ -682,11 +682,11 @@ private RelNode handleDestSchema(
             return queryRelNode;
         }
 
+        ResolvedCatalogTable resolvedCatalogTable = catalogManager.resolveCatalogTable(destTable);
```

Comment:
Please remove `catalogManager.resolveCatalogTable`, the `destTable` should always be instance of `ResolvedCatalogTable`

Reply:
forgot it, updated